### PR TITLE
Support Helm deployment with a chart

### PR DESCRIPTION
**Pull request by LeoColomb**
This change set switch static Kubernetes manifests to a Helm chart.
It should simplify both maintenance and usage.

This pull request will be done in three steps:

- [x] Initial implementation of the Helm chart
  - This to ensure we agree on the principle
- [x] Addition of GitHub Actions workflows 
  - Validate, test and release the chart
- [x] Document chart usage

### CODE REVIEW

**Comment by Kooper** [comment]
Thanks Leo! The approach looks good to me. A helm chart will help a lot with the installation and new users onboarding, so please keep up the good work :handshake:

File: README.md [hunk 1]
```diff
@@ -99,60 +99,17 @@ interactive selection of the target object.
 
 [![asciicast](https://asciinema.org/a/e2gJS70bNEQrwMXEIA64SkpR1.svg)](https://asciinema.org/a/e2gJS70bNEQrwMXEIA64SkpR1)
 
-```sh
```
Comment (Kooper) [inline README.md:102]: Do you plan to move the ssh command samples to a different place? AFAIR it is the only explanation of pod selector options available for the users, so I would like to keep it in the documentation

Comment (LeoColomb) [reply README.md:102]: Sure! These are moved to the Helm chart's NOTES.

https://github.com/LeoColomb/IngreSsh/blob/222503413231b7c84b84e96aa2d7483956737f21/charts/ingressh/templates/NOTES.txt

These notes are printed at every install and are tweaked to show only the relevant information based on the config.
They can also be re-printed, as now documented on the README.

https://github.com/LeoColomb/IngreSsh/blob/222503413231b7c84b84e96aa2d7483956737f21/README.md#connecting

**Comment by Kooper** [comment]
Thank you Leo for improving the installation and release process :+1:

**Comment by LeoColomb** [comment]
@Kooper Thanks!
I see there is a little issue with the release part (which is more complex to debug before initial run).
I'll submit a fix.

**Pull request merged**